### DISCUSSION TITLE
Connecting with ICQ possible?

### DISCUSSION THREAD

**Discussion by 82MHz** [ox/aim-oscar-server#10] [Q&A]
Hi, 
the title pretty much says it all... Old ICQ versions are using the same protocol, so is it possible to connect to your server with old ICQ clients from around 99/2000? Or would the server need to modified for this?

**Comment by pioneerman2k1** [comment]
Would be interesting. Please replay to 82MHz question

**Comment by RealBradCool** [comment]
Still abandoned
